src/scenes/GameScene.ts

```diff
@@ -8,13 +8,27 @@ const officeTilesetKey: string = 'office_tiles';
 const roomTilesetName: string = 'Room';
 const officeTilesetName: string = 'Office';
 
+const MAP: Record<string, string> = {
```

Comment:
🤔 I wonder if this should be a "configuration" json that we load at some point

What's the benefit of having a record instead of a regular dictionary?

Reply:
Record is like saying dictionary, but without inheriting a dictionary.